src/unk_02025E08.c

```diff
@@ -59,10 +57,8 @@ u16 * sub_02025E50 (SaveData * param0)
     return &v0->unk_24;
 }
 
-UnkStruct_02025E5C * sub_02025E5C (SaveData * param0)
+PlayTime *SaveData_GetPlayTime(SaveData *savedata)
 {
-    UnkStruct_02025E0C * v0;
-
-    v0 = SaveData_SaveTable(param0, 1);
-    return &v0->unk_26;
+    UnkStruct_02025E0C *player = SaveData_SaveTable(savedata, 1);
```

Comment:
why'd you label this player?

Reply:
I figured it made sense but I will change it back